src/components/slide/content/poster/poster-single.jsx

```diff
@@ -0,0 +1,539 @@
+                  value={singleSearchTypeValue}
+                  onChange={(newValue) => {
+                    setSingleSearchTypeValue(newValue);
+                  }}
+                />
+              </Col>
+            )}
+            {singleSearchType === "organizations" && (
+              <Col>
+                <label className="form-label" htmlFor="single-search-select">
+                  {t("single-search-select")}
+                </label>
+                <AsyncSelect
+                  id="single-search-select-organizations"
+                  isClearable
+                  isSearchable
+                  defaultOptions
+                  loadOptions={debounceOptions}
+                  value={singleSearchTypeValue}
+                  onChange={(newValue) => {
+                    setSingleSearchTypeValue(newValue);
+                  }}
+                />
+              </Col>
+            )}
+            {singleSearchType === "tags" && (
+              <Col>
+                <label className="form-label" htmlFor="single-search-select">
+                  {t("single-search-select")}
+                </label>
+                <AsyncSelect
+                  id="single-search-select-tags"
+                  isClearable
+                  isSearchable
+                  defaultOptions
+                  loadOptions={debounceOptions}
+                  value={singleSearchTypeValue}
+                  onChange={(newValue) => {
+                    setSingleSearchTypeValue(newValue);
+                  }}
+                />
+              </Col>
+            )}
+            <Col className="d-flex align-items-end">
+              <Button
+                onClick={singleSearchFetch}
+                className="mt-3"
+                variant="success"
+              >
+                {t("single-search-button")}
+              </Button>
+            </Col>
+          </Row>
+          <Row className="mb-2">
+            {!singleSelectedEvent && singleSearchEvents !== null && (
+              <Col>
+                <table className="table table-hover text-left">
+                  <thead>
+                    <tr>
+                      <th scope="col">{t("table-image")}</th>
+                      <th scope="col">{t("table-event")}</th>
+                      <th scope="col">{t("table-date")}</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {singleSearchEvents?.map((searchEvent) => (
+                      <tr
+                        style={{ cursor: "pointer" }}
+                        key={searchEvent.entityId}
+                        onClick={() => handleSelectEvent(searchEvent)}
+                      >
+                        <td>
+                          {searchEvent?.imageUrls?.small && (
+                            <img
+                              src={searchEvent?.imageUrls?.small}
+                              alt={searchEvent?.title}
```

Comment:
I have a hard time figuring out what this this does. click something and search something - anyways - the `alt` text is probably wrong, but perhaps you can show me the ui and explain what it does?